README.md

```diff
@@ -1,70 +1,197 @@
+### Setup
+
+Clone this repository by using the command line : git clone https://github.com/asifaminisonline/Bookstore.git
+
+### Install
+
+Install this project with:
+
+Use npm install after cloning this project
+
+### Usage
+
+- [ ] First you need to download this repo or clone this repo.
+- [ ] Open this project in your local machine.
+- [ ] Open Terminal in your machine.
+- [ ] run npm install in your terninal to install necessary required dependencies for this project.
+- [ ] Copy index.html file path and paste it in your browser url.
+- [ ] now the project will be running.
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+Github Pages.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@asifaminisonline](https://github.com/asifaminisonline)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Following are a few features i will add to the project in future.
+
+- [ ] I will add a database to this project in future where i can save user data.
+- [ ] I will add more functionality to this project like book search, author search etc.
+- [ ] I will update the interface and design of the project in future.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you're committed to making the world a better place, please support our project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to give credit to the following people who have inspired my codebase:
+
+- [ ] The Whole Microverse Team - thank you so much for this great learning platform.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]** What technologies i have used in this project ?
+
+  - [Answer_1]:Thanks for the question you can find the answer at the top section of this readme.md file where i have explained everything.
+
+- **[Question_2]** Have you used webpack in this project ?
+
+  - [Answer_2] Yes i have used webpack 5 in this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT-LICENSE.txt) licensed.
```

Comment:
- [ ] You have a link to your License file but I don't find a license file in your repository :smile: Kindly add a license file in your repository using these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to make your project licensed :+1: